packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -154,18 +163,17 @@ class ItemView extends React.Component {
       'terraClinical-ItemView',
       { 'terraClinical-ItemView--isTruncated': isTruncated },
       { [`terraClinical-ItemView--${layout}`]: layout },
-      { [`terraClinical-ItemView-accessory--${accessoryAlignment}`]: accessoryAlignment },
       customProps.className,
     ]);
 
     return (
       <div {...customProps} className={viewClassNames}>
-        {ItemView.renderAccessory(startAccessory)}
+        {ItemView.renderAccessory(startAccessory, accessoryAlignment)}
```

Comment:
I think you can conditionally render this inline with `startAccessory && ItemView.renderAccessory(startAccessory)` but what you have seems to work just fine too